pkg/server/server.go

```diff
@@ -55,7 +55,7 @@ const (
 	helmChartRepoProxyEndpoint       = "/api/helm/charts/"
 	gitopsEndpoint                   = "/api/gitops/"
 	devfileEndpoint                  = "/api/devfile/"
-	devfileSamplesEndpoint           = "/api/devfile/samples/"
+	devfileSamplesEndpoint           = "/api/devfile/samples"
```

Comment:
This is actually the fix, but when I discussed this issue with @gruselhaus, we agreed that its not required to make a `PUT` call for fetching the devfile data. So the rest of the PR changes the `PUT` Api call to `GET`. :+1: